### DISCUSSION TITLE
Create PHP 8 only version

### DISCUSSION THREAD

**Discussion by freekmurze** [spatie/laravel-multitenancy#198] [General]
Now that the package is very stable, we could opt to create a new major version that's PHP 8 only.

This would allow us to make use of PHP 8 specific features.

In my experience, this is a good idea to keep future maintenance enjoyable.

Would you be up to create a PHP 8 only version @masterix21? Breaking changes are allowed. If you don't have the time, let me know, and I'll take care of it myself in the next couple of weeks.

**Comment by masterix21** [comment]
I appreciate it, but at the moment, I'm still working on PHP 7.4, and I didn't dig inside PHP 8.0. 

What's your idea? Do you have a guideline?

**Comment by masterix21** [comment]
I bought the ebook Front Line PHP and I'm ready now. I will start the work in the next weeks.

**Comment by masterix21** [comment]
@freekmurze, I've begun to play with PHP 8.0.

https://github.com/spatie/laravel-multitenancy/tree/upgrade-to-php8

**Reply by masterix21** [reply]
What do you think about Laravel 7?  Should we remove the support using PHP 8?

**Reply by freekmurze** [reply]
Feel free to drop support for Laravel 7

**Reply by masterix21** [reply]
Great ;)

**Comment by masterix21** [comment]
I think that we are ready for the first release (v2.0.0). I will release it today, after other checks. 

Thank you, @freekmurze, for the chance and for believing in me.

**Reply by freekmurze** [reply]
Create a PR for me to check before releasing 🙂 

Looking forward to reviewing the PR 👍

**Reply by masterix21** [reply]
Created ;)
